playwright-test/tests/test_10.spec.js

```diff
@@ -0,0 +1,15 @@
+const { test } = require('../fixtures');
+const { expect } = require('@playwright/test');
+test.describe('feature foo', () => {
+  test('my test', async ({ page }) => {
+    // Assertions use the expect API.
+    await page.goto('https://www.google.com/ncr');
+    const element = await page.$('[aria-label="Search"]');
+    await element.click();
+    await element.type('BrowserStack');
```

Comment:
Use different search term in each test
for e.g. BrowserStack Playwright, BrowserStack Puppeteer, BrowserStack Cypress etc.